### PR TITLE
Gpu

### PR DESCRIPTION
**Pull request by jkatzy**

### CODE REVIEW

File: alternatives.tex [hunk 1]
```diff
@@ -1,7 +1,13 @@
 
 \subsubsection{Alternatives to Distributed machine learning}
 The most popular solution to increasing computation power for Machine
-learning is currently distributing the workload over a large amount of machines. However, there are other, more traditional ways to increase the available computation power. These ways include the use of Application Specific Integrated Circuits (ASICs) or special multi-core computer architectures.
+learning is currently distributing the workload over a large amount of machines. However, there are other, more traditional ways to increase the available computation power. Such as the use of dedicated Graphics processing units (GPUs), the use of Application Specific Integrated Circuits (ASICs) or special multi-core computer architectures.
```
Comment (mwolting) [inline alternatives.tex:4]: ". Such as" -> ", such as"

Comment (mwolting) [reply alternatives.tex:4]: "Graphics processing units" -> "Graphics Processing Units"

File: alternatives.tex [hunk 2]
```diff
@@ -1,7 +1,13 @@
...
+
+\paragraph{GPUs in Machine Learning}
+The use of GPUs in Machine Learning is already a very common method. This is due to the nature of the data which makes up Machine Learning problems. A GPU is extremly fast at making multiple small calculations on a batch of data. Luckily the data that is requiered by Machine Learning algorithms such as Neural Networks and Unsupervised Learning is of this type. This means that GPUs have had a lot of success with speeding up these calcultions as was found by Meuth \ref{Meut2007} who reported a up to 200x speed up over conventional CPUs.
```
Comment (mwolting) [inline alternatives.tex:7]: "requiered" -> "required"

Comment (mwolting) [reply alternatives.tex:7]: "calcultions" -> "calculations, "

Comment (mwolting) [reply alternatives.tex:7]: "reported a up to 200x speed up over" -> "reported a speedup factor of over 200 compared to"

File: alternatives.tex [hunk 3]
```diff
@@ -1,7 +1,13 @@
...
+
+On top of the speedup that is currently available of CPUs over GPUs, while CPUs double in processing power every 2 years, accroding to Moore's law, GPUs haev a far shorter time period in which their processing power is doubled, namely every 6 months. Which is also driven by the gaming industry which requires a lot of performance out of computers aswell \ref{Meut2007}.
```
Comment (mwolting) [inline alternatives.tex:9]: "CPUs over GPUs" -> "GPUs over CPUs"

Comment (mwolting) [reply alternatives.tex:9]: "haev" -> "have"

Comment (mwolting) [reply alternatives.tex:9]: ". Which..." -> "This development is driven by the gaming industry, which also requires high performance out of computers\ref{Meut2007}."

File: myrefs.bib [hunk 4]
```diff
@@ -41,7 +41,7 @@ @misc{Tensorflow2015
 	Andy~Davis and
 	Jeffrey~Dean and
 	Matthieu~Devin and
-	Sanjay~Ghemawat and
+	Sanjay~awat and
```
Comment (mwolting) [inline myrefs.bib:44]: ?

File: myrefs.bib [hunk 5]
```diff
@@ -76,7 +76,7 @@ @misc{Tensorflow2015
 
 @inproceedings{Tensorflow2016,
 	title = {TensorFlow: A system for large-scale machine learning},
-	author  = {Martin Abadi and Paul Barham and Jianmin Chen and Zhifeng Chen and Andy Davis and Jeffrey Dean and Matthieu Devin and Sanjay Ghemawat and Geoffrey Irving and Michael Isard and Manjunath Kudlur and Josh Levenberg and Rajat Monga and Sherry Moore and Derek G. Murray and Benoit Steiner and Paul Tucker and Vijay Vasudevan and Pete Warden and Martin Wicke and Yuan Yu and Xiaoqiang Zheng},
+	author  = {Martin Abadi and Paul Barham and Jianmin Chen and Zhifeng Chen and Andy Davis and Jeffrey Dean and Matthieu Devin and Sanjay awat and Geoffrey Irving and Michael Isard and Manjunath Kudlur and Josh Levenberg and Rajat Monga and Sherry Moore and Derek G. Murray and Benoit Steiner and Paul Tucker and Vijay Vasudevan and Pete Warden and Martin Wicke and Yuan Yu and Xiaoqiang Zheng},
```
Comment (mwolting) [inline myrefs.bib:79]: ?

File: myrefs.bib [hunk 6]
```diff
@@ -276,12 +276,12 @@ @inproceedings{Li2014Comms
 	acmid = {2968829},
 	publisher = {MIT Press},
 	address = {Cambridge, MA, USA},
-} 
+}
 
 
 
 @inproceedings{MapReduce,
-	author = {Dean, Jeffrey and Ghemawat, Sanjay},
```
Comment (mwolting) [inline myrefs.bib:284]: ?

**Review by mwolting — CHANGES_REQUESTED** [review]

File: alternatives.tex (hunk 2 above)
Comment (mwolting) [reply alternatives.tex:7]: Speedup ref is outdated, growth has stabilized probably

File: alternatives.tex (hunk 3 above)
Comment (Kloppie5) [reply alternatives.tex:9]: "accroding" -> "according"

Comment (Kloppie5) [reply alternatives.tex:9]: "aswell" -> "as well"

**Comment by Kloppie5** [comment]
PR#33 build on this and fixes its problems

**Pull request closed without merging**